sonic-utilities-tests/mock_tables/config_db.json

```diff
@@ -67,7 +67,7 @@
     "ACL_RULE|DATAACL|RULE_4": {
             "L4_SRC_PORT": "4661",
             "PACKET_ACTION": "FORWARD",
-            "PRIORITY": "9996"
+            "priority": "9996"
```

Comment:
better to create a new rule for the test than modify existing

Reply:
added a new rule in the next commit.